contributor-guide/modules/ROOT/pages/testing/boost-test-matrix.adoc

```diff
@@ -1,286 +1,178 @@
 = Boost Test Matrix
-:idprefix:
-:idseparator: -
+:navtitle: Boost Test Matrix
```

Comment:
I just went through the rendered content again. The content is quite confusing even to me who already know what these things are. 

The first thing that's confusing is that the page title is about the "Test Matrix", which has a specific meaning, but the content is mostly about individual regression/unit tests, which should already be covered in "Testing".

About the test matrices, they can also mean a generic test matrix running all regression tests, which is what we do for all libraries in CI, or it might mean the "Boost Test Matrix". Here's an example of the later:

https://www.boost.org/development/tests/master/developer/url.html

All these nuances need to be explained. The Boost Test Matrix is kind of confusing. I don't even know how it works properly. All I know is I push something to Boost.URL and I see lots of results there a few weeks later. I also know I can adjust it in

https://github.com/boostorg/url/blob/develop/meta/explicit-failures-markup.xml

but I don't really know the syntax. I just ask people when I have to. In practice, I talked to a lot of people and they all do the same. Many just completely ignore this matrix and rely on their own test matrices in CI. For instance, I wrote those cpp-actions to create the test matrix for my libraries.